skopt/space.py

```diff
@@ -10,6 +10,11 @@
 from sklearn.utils.fixes import sp_version
 
 
+class _Ellipsis:
```

Comment:
That seems like a good solution. Please add a comment to prevent people like me from asking that question again. :-)